### ISSUE TITLE
Focus trap doesn't work for a non-existent note detail dialog

### ISSUE THREAD

**Issue by codewithsyahda** [codewithsyahda/gkeep-clone-frontend-react-router#21]
When the user opens the note detail dialog with a non-existent note, the focus trap doesn't work.
